.github/PULL_REQUEST_TEMPLATE.md

```diff
@@ -0,0 +1,42 @@
+| Q | A
+| --- | ---
+| Bug fix? | [yes|no]
+| New feature? | [yes|no]
+| BC breaks? | [yes|no]
+| Deprecations? | [yes|no]
+| Tests pass? | [yes|no]
```

Comment:
I still think this line should be removed. Just look at the Symfony repository, there are plenty of PRs where this says "yes" and Travis fails. People don't think about it, so I doubt that they will question whether their PR contains tests and add them if it doesn't.

One solution would be to add "PLEASE ADD TESTS!" to the contents of this template. The problem is that people don't read the template anymore once they have seen it two or three times. It rather becomes a habit of removing pre-filled text and adding your own.

IMO what we need is someone or something that reminds them of adding the tests outside of this template. This could be as simple as commenting "Could you please add tests?" or maybe something more sophisticated, like a bot for this sort of check.